apps/earn-protocol/features/bridge/components/BridgeFormStartStep/BridgeFormStartStep.tsx

```diff
@@ -0,0 +1,262 @@
+}
+
+export const BridgeFormStartStep: FC<BridgeFormStartStepProps> = ({ state, dispatch }) => {
+  const {
+    state: { sumrNetApyConfig },
+  } = useLocalConfig()
+  const { chain: sourceChain, setChain: setSourceChain } = useChain()
+  const { userWalletAddress } = useUserWallet()
+  const sourceNetwork = chainIdToSDKNetwork(sourceChain.id)
+  const humanNetworkName = sdkNetworkToHumanNetwork(sourceNetwork)
+
+  if (!isSupportedHumanNetwork(humanNetworkName)) {
+    throw new Error('Invalid source chain')
+  }
+
+  const sumrBalanceOnSourceChain = formatToBigNumber(state.sumrBalances[humanNetworkName])
+
+  const { token: sumrToken } = useToken({
+    tokenSymbol: 'SUMMER',
+    chainId: sourceChain.id,
+  })
+  const estimatedSumrPrice = Number(sumrNetApyConfig.dilutedValuation) / SUMR_CAP
+
+  const {
+    amountRaw,
+    amountParsed,
+    amountDisplay,
+    amountDisplayUSD,
+    manualSetAmount,
+    handleAmountChange,
+    onBlur: defaultOnBlur,
+    onFocus: defaultOnFocus,
+  } = useAmount({
+    tokenDecimals: SUMR_DECIMALS,
+    tokenPrice: estimatedSumrPrice.toString(),
+    selectedToken: sumrToken,
+  })
+
+  const { checkRisk } = useRiskVerification({
+    cookiePrefix: TermsOfServiceCookiePrefix.SUMR_CLAIM_TOKEN,
+  })
+
+  const {
+    executeBridgeTransaction,
+    prepareTransaction,
+    clearTransaction,
+    transaction,
+    isPreparing,
+    isLoading,
+    error,
+    isReady,
+  } = useBridgeTransaction({
+    amount: amountRaw ?? '0',
+    destinationChain: state.destinationChain,
+    recipient: Address.createFromEthereum({ value: state.walletAddress }),
+    onSafeTxSuccess: (txHash: string) => {
+      dispatch({
+        type: 'bridge-transaction-created',
+        payload: {
+          hash: txHash,
+          amount: amountDisplay,
+        },
+      })
+    },
+    onSuccess: () => {},
+    onError: () => {},
+  })
+
+  const isSourceMatchingDestination = sourceChain.id === state.destinationChain.id
+  const isAmountGreaterThanBalance = amountParsed.gt(sumrBalanceOnSourceChain)
+
+  const validationError = isSourceMatchingDestination
+    ? 'Please select a different destination network'
+    : isAmountGreaterThanBalance
+      ? 'Insufficient balance'
+      : error?.message
+
+  const {
+    transactionFee: gasEstimate,
+    rawTransactionFee: rawGasEstimate,
+    loading: isEstimatingGas,
+  } = useGasEstimation({
+    chainId: sourceChain.id,
+    transaction: !isSourceMatchingDestination ? transaction : undefined,
+    walletAddress: Address.createFromEthereum({ value: state.walletAddress }).value,
+    overrideNetwork: sourceNetwork,
+  })
+
+  useEffect(() => {
+    if (userWalletAddress !== state.walletAddress) {
+      redirect(`/bridge/${userWalletAddress}`)
+    }
+  }, [userWalletAddress, state.walletAddress])
+
+  const handleBridge = async () => {
+    const risk = await checkRisk()
+
+    if (risk.isRisky) return
+
+    const tx = await executeBridgeTransaction()
+
+    if (!tx) return
```

Comment:
maybe it's worth to add here a console.log if tx is missing - it will be easier to debug on staging / prod if for some reason bridging wont work